src/JavascriptBuilderElement.php

```diff
@@ -176,8 +176,9 @@ public function processInternal($flowData)
         }
         
         $vars["_parameters"] = json_encode($jsParams);
-         
-        $output = $m->render(file_get_contents(__DIR__ . "/../javascript-templates/JavaScriptResource.mustache"), $vars);
+
+        $templatePath = __DIR__ . '/../vendor/51degrees/fiftyone.pipeline.javascript-templates/JavaScriptResource.mustache';
```

Comment:
what if this package is also a dependency and installed into `vendor/51degrees/fiftyone.pipeline.core` ?

Reply:
Updated.